lib/ovh.js

```diff
@@ -654,6 +663,17 @@ var https = require('https'),
     return '$1$' + crypto.createHash('sha1').update(s.join('+')).digest('hex');
   };
 
+  /**
+   * Change the custom HTTPS agent used for reaching the OVH servers
+   *
+   * @param {Object} httpsAgent: the custom HTTPS agent
+   */
+  Ovh.prototype.setCustomHTTPSAgent = function (httpsAgent) {
+    if (typeof(httpsAgent) === 'undefined' || typeof(httpsAgent) === 'object') {
+      this.httpsAgent = httpsAgent;
+    }
```

Comment:
* the typeof(httpsAgent) === 'undefined' part of the setter was meant to allow unsetting an existing agent. I've indicated it in the documentation. Perhaps there's a better way ?
* I used a setter in my application, but I could easily do without it, indeed, by injecting the agent inside the config before passing it to the constructor of the ovh module.